userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -388,9 +383,6 @@ object HelloWorld extends App {
   println("Hello world!")
 }
 {{< /tab >}}
-{{< tab header="Reference" right=true text=true >}}
```

Comment:
I agree that `www.mycplus.com` needs to be replaced or removed.

What about replacing it with `http://helloworldcollection.de`.

```
{{< /tab >}}
{{< tab header="Reference" right=true text=true >}}
{{< iframe src="http://helloworldcollection.de" >}}
{{< /tab >}}
```